pkg/storage/stores/composite_store_entry.go

```diff
@@ -39,7 +39,22 @@ type storeEntry struct {
 	stop        func()
 	fetcher     *fetcher.Fetcher
 	indexReader index.Reader
-	ChunkWriter
+	chunkWriter ChunkWriter
+}
+
+// Enforce implementation of interface
+// TODO(chaudum): Remove once interfaces are cleaned up.
```

Comment:
@chaudum what else is remaining wrt to interface clean-up?

Reply:
I think we can resolve the TODO. On the other hand, enforcing the interface does not harm. Wdyt?